brave/brave-core
# Fix TLS pins and update list of pinned roots

Resolves https://github.com/brave/brave-browser/issues/15667, https://github.com/brave/brave-browser/issues/14637, https://github.com/brave/brave-browser/issues/12349, https://github.com/brave/brave-browser/issues/2949, https://github.com/brave/brave-browser/issues/15874

Security review: https://github.com/brave/security/issues/446

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on
- [ ] Announce this change on #prod-changes

## Test Plan:

1. Open https://pinning-test.badssl.com/ and confirm that the page is blocked with a non-bypassable TLS error:
![Screenshot from 2021-05-17 11-48-45](https://user-images.githubusercontent.com/167821/118540845-df0fb480-b705-11eb-9e19-39ebaeb6407f.png)
2. If you see the following, the test has **failed**:
![Screenshot from 2021-05-07 18-48-46](https://user-images.githubusercontent.com/167821/117521860-e5af6680-af64-11eb-8d95-941fcc3ed732.png)
3. Open a new tab page and scroll down into the Brave Today section.
4. Confirm that Brave Today images are loading.
5. Open <https://creators.brave.com/> and confirm that the page loads (you dont' need to login for the test).
6. Open <https://ads-serve.brave.com/> and confirm that the page loads without a TLS error (it will display an error like `"Cannot GET /"` and that's okay).
